app/javascript/packages/verify-flow/steps/password-confirm/password-confirm-step.spec.tsx

```diff
@@ -0,0 +1,70 @@
+import sinon from 'sinon';
+import * as analytics from '@18f/identity-analytics';
+import { render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { t } from '@18f/identity-i18n';
+import { accordion } from 'identity-style-guide';
+import PasswordConfirmStep from './password-confirm-step';
+
+describe('PasswordConfirm', () => {
+  before(() => {
+    accordion.on();
+  });
+
+  after(() => {
+    accordion.off();
+  });
+
+  const sandbox = sinon.createSandbox();
+  const DEFAULT_PROPS = {
+    onChange() {},
+    onError() {},
+    errors: [],
+    toPreviousStep() {},
+    registerField: () => () => {},
+    unknownFieldErrors: [],
+    value: {},
+  };
+
+  beforeEach(() => {
+    sandbox.spy(analytics, 'trackEvent');
+  });
+
+  afterEach(() => {
+    sandbox.restore();
+  });
+
+  it('it has a collapsed accordion by default', () => {
+    const toPreviousStep = sinon.spy();
+    const { getByText } = render(
+      <PasswordConfirmStep {...DEFAULT_PROPS} toPreviousStep={toPreviousStep} />,
+    );
```

Comment:
Minor, and also throughout the file: We're not doing anything with the `toPreviousStep` spy, so there's not much value in passing it in as a prop as a replacement to the default value.

```suggestion
    const { getByText } = render(<PasswordConfirmStep {...DEFAULT_PROPS} />);
```